lib/client/db/dbcmd/error.go

```diff
@@ -44,6 +44,16 @@ func ConvertCommandError(cmd *exec.Cmd, err error, peakStderr string) error {
 				redisBin,
 			)
 		}
+	case mysqlBin, mariadbBin:
+		// Connection failure.
+		if strings.HasPrefix(peakStderr, "ERROR 1105 (HY000):") {
```

Comment:
The `1105`-  `HY000` errors code is marked as generics/unknown error. Not sure if we should freely map all those errors to `AccessDenied`

Reply:
I agree it might not be the best, but since it is the first error that the CLI returns (as we are peeking the stderr), it will probably be a connectivity issue. As I proposed in the comment below, we could change this error to `ConnectionProblem`.